mmdet/evaluation/metrics/openimages_metric.py

```diff
@@ -1,237 +1,185 @@
-        # get dataset type
-        dataset_type = self.dataset_meta.get('dataset_type')
-        if dataset_type not in ['oid_challenge', 'oid_v6']:
-            dataset_type = 'oid_v6'
-            print_log(
-                'Cannot infer dataset type from the length of the'
-                ' classes. Set `oid_v6` as dataset type.',
-                logger='current')
-        mean_aps = []
         for i, (iou_thr,
-                ioa_thr) in enumerate(zip(self.iou_thrs, self.ioa_thrs)):
-            if self.use_group_of:
-                assert ioa_thr is not None, 'ioa_thr must have value when' \
-                                            ' using group_of in evaluation.'
-            print_log(f'\n{"-" * 15}iou_thr, ioa_thr: {iou_thr}, {ioa_thr}'
-                      f'{"-" * 15}')
-            mean_ap, _ = eval_map(
-                preds,
-                gts,
-                scale_ranges=self.scale_ranges,
-                iou_thr=iou_thr,
-                ioa_thr=ioa_thr,
-                dataset=dataset_type,
-                logger=logger,
-                use_group_of=self.use_group_of)
-
-            mean_aps.append(mean_ap)
-            eval_results[f'AP{int(iou_thr * 100):02d}'] = round(mean_ap, 3)
-        eval_results['mAP'] = sum(mean_aps) / len(mean_aps)
-        return eval_results
+                iof_thr) in enumerate(zip(self.iou_thrs,
+                                          self.iof_thrs)):  # noqa: E501
+            for j, scale_range in enumerate(self.scale_ranges):
+                table_title = ' Bbox Results ' \
+                              f'(IoU thr={iou_thr}, IoF thr={iof_thr}) '
+                if scale_range != (None, None):
+                    table_title += f'Scale range: {scale_range} '
+
+                table_data = [header]
+                aps = []
+                for k in range(len(classes)):
+                    class_results = classwise_result[k]
+                    recalls = class_results['recalls'][i, j]
+                    recall = 0 if len(recalls) == 0 else recalls[-1]
+                    row_data = [
+                        classes[k], class_results['num_gts'][i, j],
+                        class_results['num_dets'],
+                        round(recall * 100, 2),
+                        round(class_results['ap'][i, j] * 100, 2)
+                    ]
+                    table_data.append(row_data)
+                    if class_results['num_gts'][i, j] > 0:
+                        aps.append(class_results['ap'][i, j])
+
+                mean_ap = np.mean(aps) if aps != [] else 0
+                table_data.append(
+                    ['mAP', '', '', '', f'{round(mean_ap * 100, 2)}'])
+                table = AsciiTable(table_data, title=table_title)
+                table.inner_footing_row_border = True
+                print_log('\n' + table.table, logger='current')
+
+        evaluate_results = {
+            f'openimages/{k}(%)': round(float(v) * 100, 4)
```

Comment:
Why not use prefix?

Reply:
It is not supported in mmeval, I have proposed a PR: https://github.com/open-mmlab/mmeval/pull/99/